test/rpc/cart_rpc_one_node.yaml

```diff
@@ -36,6 +36,7 @@ tests: !mux
     cli_ppn: "1"
   echo:
     name: echo_basic
+    logparse: "1"
```

Comment:
Does yaml have a boolean type?

Reply:
Don't really know as this is how its done elsewhere as well. Do you have a suggestion on how it should be done. It may have to be incorporated elsewhere too to keep consistency.

Reply:
It looks like you can just set this to Yes/No

https://yaml.org/refcard.html

Reply:
I have made an associated change in cart_utils.py as well to accommodate for the Yes/No usage. Please review that as well on my next push.